src/tests/api/utils/ApiTestSteps.ts

```diff
@@ -65,19 +77,60 @@ export async function getTxmaSqsEvent(): Promise<any> {
 	return sqsClient.send(command);
 }
 
-export async function getCompletedDynamoRecord(user: string): Promise<GetItemCommandOutput> {
-	const command = new GetItemCommand({
-		Key: {
-			userId: { "S": user },
-		},
-		TableName: SESSION_EVENTS_TABLE,
-	});
-	console.log(command);
-	let response: GetItemCommandOutput;
+export async function getSessionByUserId(userId: string, tableName: string): Promise<SessionEvent | undefined> {
+	interface OriginalValue {
+		N?: string;
+		S?: string;
+		L?: string;
+		BOOL?: boolean;
+	}
+	
+	interface OriginalSessionItem {
+		[key: string]: OriginalValue;
+	}
+
+	let session;
+	let response;
+
+	try {
+		do {
+			response = await HARNESS_API_INSTANCE.get<{ Item: OriginalSessionItem }>(`getRecordByUserId/${tableName}/${userId}`, {});
+		} while (!(response.data.Item?.notified && response.data.Item.notified.BOOL));
+		const originalSession = response.data.Item;
+
+		session = Object.fromEntries(
+			Object.entries(originalSession).map(([key, value]) => [key, value.N ?? value.S ?? value.L ?? value.BOOL]),
+		) as unknown as SessionEvent;
+	} catch (e: any) {
+		console.error({ message: "getSessionByUserId - failed getting session from Dynamo", e });
+	}
+
+	console.log("getSessionByUserId Response", session);
+	return session;
+}
+
+export async function getSqsEventList(folder: string, prefix: string, txmaEventSize:number): Promise<any> {
+	let keys: any[];
+	let keyList: any[];
+	let i:any;
 	do {
-		response = await dynamoClient.send(command);
-	} while (!(response.Item?.notified && response.Item.notified.BOOL));
-	return response;
+		const listObjectsResponse = await HARNESS_API_INSTANCE.get("/bucket/", {
+			params: {
+				prefix: folder + prefix,
+			},
+		});
```

Comment:
just to check - if calling this in a do...while loop to iterate through the pages the second and subsequent calls need to know where to start from?

Reply:
Method was incorrectly copied over from another BE repo (CIC or F2F) and is not in use. Has been removed.